bindata/v4.0.0/apiservice-cabundle-controller/defaultconfig.yaml

```diff
@@ -1,7 +1,3 @@
 apiVersion: servicecertsigner.config.openshift.io/v1alpha1
 kind: APIServiceCABundleInjectorConfig
 caBundleFile: /var/run/configmaps/signing-cabundle/ca-bundle.crt
-authentication:
```

Comment:
@stlaz Is this just a configuration change that prompts library code to ensure the metrics endpoint is secured? If so, why would that require operator-specific e2e coverage?

Reply:
It would require a library extended test, or at least an integration one, but we would still want to check that the operator is actually behaving the intended way and publishes it's metrics as desired, no?

Reply:
I finally got to browse through the code - it was not my point to try to check that the endpoints are authenticated, but that they are actually showing something

Reply:
Couldn't that be validated in a unit test?

Reply:
I think that that would be a fine general approach to test an endpoint, yes. I wonder how many endpoints we'd like to test this way though.

Reply:
I've identified 10 operators in the service ca compatibility audit that should be tested this way, and there are likely more.

Reply:
I think what @marun suggested makes sense to me.